neo/Ledger/ContractState.cs

```diff
@@ -57,15 +56,15 @@ ContractState ICloneable<ContractState>.Clone()
         public override void Deserialize(BinaryReader reader)
         {
             base.Deserialize(reader);
-            Script = reader.ReadVarBytes();
-            ParameterList = reader.ReadVarBytes().Select(p => (ContractParameterType)p).ToArray();
+            Script = reader.ReadVarBytes(1024 * 1024);
+            ParameterList = reader.ReadVarBytes(252).Select(p => (ContractParameterType)p).ToArray();
             ReturnType = (ContractParameterType)reader.ReadByte();
             ContractProperties = (ContractPropertyState)reader.ReadByte();
-            Name = reader.ReadVarString();
-            CodeVersion = reader.ReadVarString();
-            Author = reader.ReadVarString();
-            Email = reader.ReadVarString();
-            Description = reader.ReadVarString();
+            Name = reader.ReadVarString(252);
+            CodeVersion = reader.ReadVarString(252);
+            Author = reader.ReadVarString(252);
+            Email = reader.ReadVarString(252);
```

Comment:
why 252?  Is 255 better, like the default varchar length in mysql?

Reply:
Got, the remaining 4 bytes represent the var length

Reply:
The syscall has this same values

Reply:
👍, thanks